bears/python/PyFlakesBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='pyflakes',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>.*):\s*'
+                     r'(?P<severity>)\s(?P<message>.*)',
+        severity_map={
+            '': RESULT_SEVERITY.INFO
+        })
+class PyFlakesBear:
+    """
+    Checks Python files for logistic errors using ``pyflakes``.
```

Comment:
"erroneous and ambiguous syntax" covers pyflakes pretty accurately. Ive not seen a clearer definition. The maintainers have a goal of only reporting errors, but that doesnt explain 50% of the messages.